frontend/src/openapi/v2/core/request.ts

```diff
@@ -225,12 +225,18 @@ function catchErrors(options: ApiRequestOptions, result: ApiResult): void {
         ...options.errors,
     }
 
-    const error = errors[result.status];
-    if (error) {
-        throw new ApiError(result, error);
-    }
-
     if (!result.ok) {
```

Comment:
This file is automatically generated... i don't think you can directly modify it....